test-automation/api-tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+.note
```

Comment:
new .gitignore not needed, please add to rootmost one: https://github.com/harmony-one/harmony-ops/blob/master/.gitignore